tests/Geometry/Polygon.py

```diff
@@ -0,0 +1,56 @@
+from pycam.Geometry.Polygon import *
+from pycam.Geometry.Line import *
```

Comment:
Alright, dagnabbit, I am going to get to the root of this relative import shenaniganry.   It should "Just Work".